util/session/sessionmanager_test.go

```diff
@@ -4,26 +4,31 @@ import (
 	"context"
 	"testing"
 
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/argoproj/argo-cd/common"
+
 	"github.com/dgrijalva/jwt-go"
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/fake"
 
 	"github.com/argoproj/argo-cd/errors"
-	"github.com/argoproj/argo-cd/util/password"
+	p "github.com/argoproj/argo-cd/util/password"
 	"github.com/argoproj/argo-cd/util/settings"
 )
 
-func TestSessionManager(t *testing.T) {
-	const (
-		defaultSecretKey = "Hello, world!"
-		defaultSubject   = "argo"
-	)
+const password = "password"
```

Comment:
does this constant have to be global, or can it be localized to a method like the pattern used by other constants?